presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -54,6 +54,26 @@ namespace facebook::presto {
 
 namespace {
 
+// Repeatedly calls for cleanOldTasks() for a while to ensure that we overcome a
```

Comment:
for cleanOldTasks() -->= cleanOldTasks()

Reply:
Thanks!
Let's fix this some other time.
Now it is important to land this to remove the noise.